docs/data-pipeline.md

```diff
@@ -160,3 +172,5 @@ The data version is expected to be a well-defined, namespaced identifier for the
 - `FOO_1_1` → Version 1.1 of data from a hypothetical data source named Foo
 
 The first data version subtag, or namespace, defines the syntax for the remainder of the identifier.  For example, the `CLDR` namespace might accept two or three subtags: major version (`37`), minor version (`alpha1`), and optional patch version (`goog2020a`).
+
+*Note:* The syntax for the data version is undefined at this time.  What is shown above is merely a strawman example.
```

Comment:
Glad that this was noted as such.  FWIW, after I saw [this talk](https://www.youtube.com/watch?v=oyLBGkS5ICk) on versioning, I started to think differently about semantic versioning (not assuming it to the be standard / best way) and to think about compatibility and versioning of code differently (or at least be more cautious & considerate).  But that's just my current perspective.